[ALLI-7835] Update Finnish Public Libraries Classification System translations

aleksip opened the pull request.

aleksip committed:
[ALLI-7835] Update Finnish Public Libraries Classification System translations

karisera committed:
Update en-gb.ini

karisera committed:
Update fi.ini

karisera committed:
Update sv.ini

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -242,22 +243,23 @@ ykl 33.5 = "Rikosoikeus"
 ykl 34.1 = "Valtakunnansuunnittelu"
 ykl 34.2 = "Seutusuunnittelu"
-ykl 34.5 = "Yhdyskuntasuunnittelu. Kaupunkisuunnittelu Ympäristönsuunnittelu"
+ykl 34.5 = "Yhdyskuntasuunnittelu. Kaupunkisuunnittelu ympäristönsuunnittelu"
```
Luulen, että tämän pitäisi olla `Yhdyskuntasuunnittelu. Kaupunkisuunnittelu. Ympäristönsuunnittelu`. Laitoin kysymyksen Finton suuntaan, mutta korjaisin sen tähän. Muilla kielillä onkin niin.

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -1184,7 +1187,7 @@ ykl 78.48133 = "Indonesia. Malesia. Brunei"
 ykl 78.4814 = "Keski-Aasia"
 ykl 78.48141 = "Mongolia"
-ykl 78.48142 = "Kazakstan. Kirgisia. Tadzikistan. Turkmenistan Uzbekistan"
+ykl 78.48142 = "Kazakstan. Kirgisia. Tadzikistan. Turkmenistan uzbekistan"
```
Tässä pitäisi nähdäkseni myös Uzbekistan olla isolla ja piste ennen sitä.

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -1441,16 +1444,16 @@ ykl 78.916 = "Romantiikan ajan musiikki (n. 1820-1900)"
 ykl 78.92 = "Suomen musiikin historia"
 ykl 78.93 = "Pohjoismaiden musiikin historia"
-ykl 78.94 = "Keski-Euroopan maiden musiikin historia"
+ykl 78.94 = "Keski-euroopan maiden musiikin historia"
```
Euroopan isolla alkukirjaimella.

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -1441,16 +1444,16 @@ ykl 78.916 = "Romantiikan ajan musiikki (n. 1820-1900)"
 ykl 78.95 = "Brittein saarten musiikin historia"
-ykl 78.96 = "Espanjan, Italian, Portugalin ja Ranskan musiikin historia"
-ykl 78.97 = "Muiden Euroopan maiden musiikin historia"
+ykl 78.96 = "Espanjan, italian, portugalin ja ranskan musiikin historia"
```
Isot alkukirjaimet.

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -1441,16 +1444,16 @@ ykl 78.916 = "Romantiikan ajan musiikki (n. 1820-1900)"
-ykl 78.96 = "Espanjan, Italian, Portugalin ja Ranskan musiikin historia"
-ykl 78.97 = "Muiden Euroopan maiden musiikin historia"
+ykl 78.96 = "Espanjan, italian, portugalin ja ranskan musiikin historia"
+ykl 78.97 = "Muiden euroopan maiden musiikin historia"
```
Euroopan isolla.

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -1983,8 +2002,8 @@ ykl 86.43 = "Friisinkielisen kirjallisuuden historia ja tutkimus"
 ykl 86.53 = "Amerikan englanninkielisen kirjallisuuden historia ja tutkimus"
-ykl 86.54 = "Australian ja Uuden-Seelannin englanninkielisen kirjallisuuden historia ja tutkimus"
-ykl 86.56 = "Etelä-Afrikan englanninkielisen kirjallisuuden historia ja tutkimus"
+ykl 86.54 = "Australian ja uuden-seelannin englanninkielisen kirjallisuuden historia ja tutkimus"
```
Isot alkukirjaimet.

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -1983,8 +2002,8 @@ ykl 86.43 = "Friisinkielisen kirjallisuuden historia ja tutkimus"
-ykl 86.54 = "Australian ja Uuden-Seelannin englanninkielisen kirjallisuuden historia ja tutkimus"
-ykl 86.56 = "Etelä-Afrikan englanninkielisen kirjallisuuden historia ja tutkimus"
+ykl 86.54 = "Australian ja uuden-seelannin englanninkielisen kirjallisuuden historia ja tutkimus"
+ykl 86.56 = "Etelä-afrikan englanninkielisen kirjallisuuden historia ja tutkimus"
```
Isot alkukirjaimet.

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -2192,10 +2213,10 @@ ykl 93.14 = "V. 1810-1905"
 ykl 93.4 = "Islanti"
 ykl 93.5 = "Pohjoiskalotti"
-ykl 94 = "KESKI-EUROOPAN HISTORIA"
+ykl 94 = "Keski-euroopan historia"
```
Iso alkukirjain

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -2211,15 +2232,15 @@ ykl 94.8 = "Benelux-maat"
 ykl 95.71 = "Irlannin tasavalta"
 ykl 95.72 = "Pohjois-Irlanti"
-ykl 96 = "ETELÄ-EUROOPAN HISTORIA"
+ykl 96 = "Etelä-euroopan historia"
```
Iso alkukirjain

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -2235,33 +2256,33 @@ ykl 96.5 = "Malta"
 ykl 96.84 = "Albania"
 ykl 96.85 = "Kreikka"
-ykl 97 = "ITÄ-EUROOPAN HISTORIA"
+ykl 97 = "Itä-euroopan historia"
```
Iso alkukirjain

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -2385,7 +2406,7 @@ udkf 21 = "Luonnollinen teologia. Vapaa-ajattelu. Ateismi. Panteismi"
 udkf 22 = "Raamattu"
 udkf 23 = "Dogmatiikka"
-udkf 24 = "Kristillinen etiikka. Kristillinen taide. Hymnologia. Hartaus. Mystiikka. Askeesi Uskonnolliset kokemukset"
+udkf 24 = "Kristillinen etiikka. Kristillinen taide. Hymnologia. Hartaus. Mystiikka. Askeesi uskonnolliset kokemukset"
```
Pitäisiköhän tässäkin olla `Askeesi. Uskonnolliset..`?

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -2507,7 +2528,7 @@ udkf 53 = "Fysiikka"
 udkf 533 = "Kaasujen mekaniikka. Aerodynamiikka"
 udkf 534 = "Mekaaniset värähtelyt. Akustiikka"
-udkf 535 = "Valo-oppi Optiikka"
+udkf 535 = "Valo-oppi optiikka"
```
Piste ja isolla?

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -2557,7 +2578,7 @@ udkf 614.2 = "Lääkärit. Sairaanhoitohenkilökunta. Terveydenhuoltohenkilöst
 udkf 614.8 = "Tapaturmantorjunta. Säteilyturvallisuus. Pelastuspalvelu. Ensiapu"
 udkf 614.84 = "Palontorjunta"
-udkf 615 = "Farmakologia Farmasia"
+udkf 615 = "Farmakologia farmasia"
```
Piste ja isolla?

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -2566,7 +2587,7 @@ udkf 615.851 = "Psykoterapia. Psykoanalyysi"
 udkf 616-002 = "Tuberkuloosi. Reumataudit"
 udkf 616-006 = "Onkologia. Kasvaimet. Syöpätaudit"
-udkf 616-053.2 = "Pediatria Lastentaudit"
+udkf 616-053.2 = "Pediatria lastentaudit"
```
Piste ja isolla?

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -2578,7 +2599,7 @@ udkf 616.24 = "Keuhkosairaudet"
 udkf 616.314 = "Odontologia. Hammaslääketiede. Hammastaudit"
 udkf 616.33/.38 = "Gastroenterologia. Mahan ja suoliston taudit"
-udkf 616.379-008.64 = "Diabetes Sokeritauti"
+udkf 616.379-008.64 = "Diabetes sokeritauti"
```
Piste ja isolla?

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -2596,14 +2617,14 @@ udkf 620.1 = "Aineenkoetus"
 udkf 620.22 = "Materiaalioppi"
 udkf 620.9 = "Energiatalous. Energiapolitiikka"
-udkf 621 = "Konetekniikka Mekaaninen teknologia"
+udkf 621 = "Konetekniikka mekaaninen teknologia"
```
Piste ja isolla?

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -2596,14 +2617,14 @@ udkf 620.1 = "Aineenkoetus"
 udkf 621.31 = "Voimalat. Sähköenergia. Soveltava sähkömagnetismi. Soveltava sähköstatiikka. Sähkökoneet. Sähkömittaustekniikka"
 udkf 621.37/.38 = "Elektroniikka"
-udkf 621.38 = "Röntgentekniikka Bioelektroniikka"
+udkf 621.38 = "Röntgentekniikka bioelektroniikka"
```
Piste ja isolla?

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -2675,7 +2696,7 @@ udkf 656.835 = "Postimerkkeily. Filatelia"
 udkf 658 = "Liiketalous"
 udkf 658.2 = "Yrityksen sijainti. Yrityksen tilat. Tuotantovälineet. Kuljetukset"
-udkf 658.3 = "Henkilöstöhallinto Henkilöstölehdet & ala"
+udkf 658.3 = "Henkilöstöhallinto henkilöstölehdet & ala"
```
Piste ja isolla?

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -2695,7 +2716,7 @@ udkf 664.6/.7 = "Viljateollisuus. Leipomot. Myllyt"
 udkf 666.1 = "Lasiteollisuus"
 udkf 666.3/.7 = "Keraaminen teollisuus"
-udkf 666.9 = "Sementtiteollisuus Betoniteollisuus. Kipsiteollisuus. Kalkkiteollisuus"
+udkf 666.9 = "Sementtiteollisuus betoniteollisuus. Kipsiteollisuus. Kalkkiteollisuus"
```
Piste ja isolla?

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -2731,7 +2752,7 @@ udkf 741.5 = "Sarjakuvat"
 udkf 77 = "Valokuvaus"
 udkf 78 = "Musiikki"
-udkf jaottelu = "käyttämällä luokkaa 78 ja sopivaa muuta luokkaa:"
+udkf jaottelu = "Käyttämällä luokkaa 78 ja sopivaa muuta luokkaa:"
```
Tämä taitaa olla höpörivi, jonka voisi jättää pois.

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -2757,7 +2778,7 @@ udkf 80 = "Kielet. Filologia"
 udkf 800.89 = "Keinotekoiset kielet. Esperanto. Elekielet. Viittomakieli"
 udkf 801.3 = "Sanastontutkimus. Leksikologia. Nimistöntutkimus. Onomastiikka"
-udkf 801.4 = "Äänneoppi Fonetiikka. Fonologia"
+udkf 801.4 = "Äänneoppi fonetiikka. Fonologia"
```
Piste ja isolla?

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -3357,7 +3378,7 @@ udk2 524.1 = "Kosminen säteily. Ensisijainen kosminen säteily"
 udk2 524.4 = "Tähtijoukot ja tähtiassosiaatiot"
 udk2 524.5 = "Tähtienvälinen aine. Galaksit"
-udk2 524.6 = "Galaksi (Linnunrata)"
+udk2 524.6 = "Galaksi (linnunrata)"
```
Linnunrata isolla

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -3532,7 +3553,7 @@ udk2 550.3 = "Geofysiikka"
 udk2 550.7 = "Geobiologia. Organismien geologinen vaikutus"
 udk2 550.8 = "Malminetsintä. Soveltava geologia"
-udk2 550.93 = "Absoluuttisen geologisen iän määrittäminen. Iän määritysmenetelmät"
+udk2 550.93 = "Absoluuttisen geologisen Iän määrittäminen. Iän määritysmenetelmät"
```
Ensimmäinen Iän pitäisi olla pienellä.

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -3575,7 +3596,7 @@ udk2 57.01 = "Yleiset lait. Teoreettiset näkökohdat ja ominaisuudet"
 udk2 57.03 = ""
 udk2 57.04 = "Tekijät. Vaikutukset"
-udk2 57.05 = "Säätelyprosessien periaatteet eri tasoilla, esim. molekyyli- ja solutasolla, jne."
+udk2 57.05 = "Säätelyprosessien periaatteet eri tasoilla, esim. Molekyyli- ja solutasolla, jne."
```
molekyyli pienellä.

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -3791,19 +3812,19 @@ udk2 599.742.7 = "Felidae (kissaeläimet)"
 udk2 599.818 = ""
 udk2 599.82/.89 = "Simiiformes. Apinat"
-udk2 599.82 = "Platyrrhini (Leveänenäapinat )"
+udk2 599.82 = "Platyrrhini (leveänenäapinat )"
```
Ylimääräinen välilyönti ennen loppusulkua

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -3684,7 +3705,7 @@ udk2 594 = "Mollusca. Nilviäiset. Brachiopoda. Lonkerojalkaiset"
 udk2 595.7 = "Entomologia. Hyönteiset"
 udk2 596/599 = "Chordata. Selkäjänteiset"
-udk2 596.2 = "Tunicata /Urochordata (vaippaeläimet)"
+udk2 596.2 = "Tunicata /urochordata (vaippaeläimet)"
```
Lisättävä välilyönti kauttaviivan perään.

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -3864,7 +3885,7 @@ udk2 616-01/-099 = "Yleisiä patologiaa koskevia näkökohtia"
 udk2 616-02 = "Etiologia. Taudinsyyt. Taudinaiheuttajat."
 udk2 616-03 = "Erilaiset tautimuodot ja niiden kehitys"
-udk2 616-05 = "Sairaan persoonallisuus. (Yksilölliset ja ympäristötekijät)"
+udk2 616-05 = "Sairaan persoonallisuus. (yksilölliset ja ympäristötekijät)"
```
Olisi varmaan parempi ilman pistettä

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -4269,7 +4290,7 @@ udk2 684.6 = "Taidepuusepäntyöt. Vanerilla päällystäminen. Upotustyöt. Int
 udk2 685 = "Nahkatyöt. Naahkavarusteet. Matkailu-, urheilu-, ja viihdevälineet"
 udk2 685.1 = "Satulat. Suitset. Kannukset"
-udk2 685.2 = "Erilaiset nahka- ja kangasvarusteet, esim. vyöt, nahkapuserot jne."
+udk2 685.2 = "Erilaiset nahka- ja kangasvarusteet, esim. Vyöt, nahkapuserot jne."
```
vyöt pienellä

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -4291,12 +4312,12 @@ udk2 689 = "Tee-se-itsetyöt. Käsityöt"
 udk2 691 = "Rakennusaineet. Rakennustarvikkeet"
 udk2 692 = ""
-udk2 693 = "Muuraus. Betonivalu ym. rakennustyöt"
+udk2 693 = "Muuraus. Betonivalu ym. Rakennustyöt"
```
rakennustyöt pienellä

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -4291,12 +4312,12 @@ udk2 689 = "Tee-se-itsetyöt. Käsityöt"
+udk2 693 = "Muuraus. Betonivalu ym. Rakennustyöt"
 udk2 694 = "Puurakenteet. Kirvesmiehentyöt. Rakennuspuusepäntyöt"
-udk2 696 = "LVI-asennukset. Sähköasennukset"
+udk2 696 = "Lvi-asennukset. Sähköasennukset"
```
LVI isolla

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -4309,7 +4330,7 @@ udk2 7.035 = "Post-renessanssi"
 udk2 7.037 = ""
 udk2 7.038 = "Abstrakti taide. Ei-esittävä taide"
-udk2 7.04 = "Taideteoksen aihe. lkonografia. Ikonologia"
+udk2 7.04 = "Taideteoksen aihe. Lkonografia. Ikonologia"
```
Pitäisikö olla Ikonografia, iillä?

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -4479,9 +4500,9 @@ udk2 798.2 = "Ratsastus"
 udk2 798.6 = ""
 udk2 798.8 = "Koirakilpailut"
-udk2 798.9 = "Muiden eläinten kilpailut, esim. linnut"
+udk2 798.9 = "Muiden eläinten kilpailut, esim. Linnut"
```
linnut pienellä

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -4693,7 +4714,7 @@ udk2 821.6 = ""
 udk2 821.8 = ""
 udk2 821.9 = ""
-udk2 9 = "ARKEOLOGILINEN KIRJALLISUUS. ESIHISTORIA. KOTISEUTUOPPI. MAANTIEDE. HISTORIA. ELÄMÄKERRAT"
+udk2 9 = "Arkeologilinen kirjallisuus. Esihistoria. Kotiseutuoppi. Maantiede. Historia. Elämäkerrat"
```
Pitäisiköhän tämän olla Arkeologinen?

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -42,39 +42,39 @@ ykl 02.6 = "Barnbiblioteksarbete. Ungdomsbiblioteksarbete"
 ykl 07.22 = "Programverksamhet i television"
-ykl 1 = "FILOSOFI. PSYKOLOGI PARANORMALA FENOMEN"
-ykl 10 = "ALLMÄNNA VERK"
+ykl 1 = "Filosofi. Psykologi paranormala fenomen. Vetenskapsteori"
```
Piste ennen ja `Paranormala` isolla alkukirjaimella.

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -728,13 +731,14 @@ ykl 65.8 = "Leksaker"
 ykl 66.32 = "Konstruktionsdelar"
 ykl 66.33 = "Byggmaterial. Byggnadsarbeten"
-ykl 66.34 = "VVS-arbeten. Elarbeten"
+ykl 66.34 = "Vvs-arbeten. Elarbeten"
```
VVS isolla

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -1294,7 +1298,7 @@ ykl 78.5589 = "Andra konserter för ett blåsinstrument"
 ykl 78.6 = "Musik för tangentinstrument. Musik för knäppinstrument"
 ykl 78.61 = "Piano"
-ykl 78.611 = "Musik är avsedd som studiematerial i pianospel (t.ex. etyder)"
+ykl 78.611 = "Musik är avsedd som studiematerial i pianospel (t.Ex. Etyder)"
```
Ex ja Etyder pienellä

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -1295,7 +1298,7 @@ ykl 78.5589 = "Muut konsertot yhdelle puhallinsoittimelle"
 ykl 78.6 = "Kosketinsoitinmusiikki. Näppäilysoitinmusiikki"
 ykl 78.61 = "Piano"
-ykl 78.611 = "Pianonsoiton opintoaineistoksi tarkoitettu musiikki (esim. etydit)"
+ykl 78.611 = "Pianonsoiton opintoaineistoksi tarkoitettu musiikki (esim. Etydit)"
```
etydit pienellä

EreMaijala commented on local/languages/finna/classification/en-gb.ini:
```diff
@@ -1293,7 +1298,7 @@ ykl 78.5589 = "Other concertos for one wind instrument"
 ykl 78.6 = "Keyboard music. Music for plucked instruments"
 ykl 78.61 = "Piano"
-ykl 78.611 = "Music intended as study material for piano playing (e.g. études)"
+ykl 78.611 = "Music intended as study material for piano playing (e.g. Études)"
```
Études pienellä

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -1986,7 +2006,7 @@ ykl 86.54 = "Litteraturhistoria och -forskning som rör litteratur på engelska
 ykl 86.6 = "Litteraturhistoria och -forskning som rör litteratur på romanska språk"
 ykl 86.61 = "Litteraturhistoria och -forskning som rör litteratur på franska"
-ykl 86.62 = "Litteraturhistoria och -forskning som rör litteratur på Italienska"
+ykl 86.62 = "Litteraturhistoria och -forskning som rör litteratur på italienska"
```
Italienska isolla

karisera replied:
mä veikkaan että tossa on joku fiba koska ei muutkaan kielet tule ruotsiksi isolla?

EreMaijala replied:
hups, joo, väärin väitin

EreMaijala commented on local/languages/finna/classification/fi.ini:
```diff
@@ -404,15 +406,15 @@ ykl 44.8 = "Benelux-maat"
 ykl 44.82 = "Alankomaat"
 ykl 44.83 = "Luxemburg"
-ykl 45 = "BRITTEIN SAARET"
+ykl 45 = "Brittein saaret"
```
Britteinsaaret yhdyssana.

karisera replied:
Siitä pitää sit ehkä laittaa palautetta YKL-ylläpitoon, mutta korjaan nyt tuon

EreMaijala replied:
Ei korjattu, pitikö?

EreMaijala replied:
Ei pitänyt.

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -2375,7 +2397,7 @@ udk2 004.2 = "Dataarkitektur"
 udk2 004.23 = "Instruktionsuppsättnings arkitektur"
 udk2 004.25 = "Dataminne. Cacheminne"
-udk2 004.27 = "Avancerade arkitekturer. Icke-Von Neumann arkitekturer"
+udk2 004.27 = "Avancerade arkitekturer. Icke-von neumann arkitekturer"
```
`Von Neumann` isolla?

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -2595,23 +2617,23 @@ udk2 233 = "Hinduism i strikt mening"
 udk2 25 = "Antikens religioner"
 udk2 251 = "Fornegyptisk religion"
-udk2 252 = "Religioner i Mesopotamien"
+udk2 252 = "Religioner i mesopotamien"
```
mesopotamien isolla

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -2595,23 +2617,23 @@ udk2 233 = "Hinduism i strikt mening"
 udk2 255 = "Grekisk och romersk religion"
 udk2 257 = "Europeiska religioner"
-udk2 258 = "Religioner i Syd- och Centralamerica. Förkolombianska Indianska religioner"
+udk2 258 = "Religioner i syd- och centralamerica. Förkolombianska indianska religioner"
```
Syd ja centralamerica isolla

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -2819,8 +2841,8 @@ udk2 365 = "Hjälp med bostad"
 udk2 37 = "Utbildning"
-udk2 37.01/.09 = "Speciella tillägg för teori, principer.metoder och organisation av utbildningen"
-udk2 37.01 = "Pedagogiska grundvalar. Teori. Policy m.m."
+udk2 37.01/.09 = "Speciella tillägg för teori, principer.Metoder och organisation av utbildningen"
```
Pisteen jälkeen väli

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -2877,12 +2899,12 @@ udk2 524.1 = "Kosmisk strålning. Primär kosmisk strålning"
 udk2 524.4 = "Stjärnhopar. Sammanslutningar av stjärnor"
 udk2 524.5 = "Det interstellära mediet. Nebulosor"
-udk2 524.6 = "Galaxen (Vintergatan)"
+udk2 524.6 = "Galaxen (vintergatan)"
```
vintergatan isolla

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -3125,29 +3147,29 @@ udk2 581.9 = "Geografisk botanik. Växtgeografin (fytogeografi). Floror. Växter
-udk2 582.097 = "Trädklättrare (t.ex. murgröna - araliaceae)"
-udk2 582.099 = "Örtartade eller icke-vedartade växter (ofta kallade "plantor" eller "blommor")"
+udk2 582.093 = "Buskar (mindre vedartade växter utan central+f1393stam)"
+udk2 582.095 = "Undervegetation (t.Ex. Björnbär - rosoideae)"
```
Ex. Björnbär pienillä

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -3125,29 +3147,29 @@ udk2 581.9 = "Geografisk botanik. Växtgeografin (fytogeografi). Floror. Växter
-udk2 582.099 = "Örtartade eller icke-vedartade växter (ofta kallade "plantor" eller "blommor")"
+udk2 582.093 = "Buskar (mindre vedartade växter utan central+f1393stam)"
+udk2 582.095 = "Undervegetation (t.Ex. Björnbär - rosoideae)"
+udk2 582.097 = "Trädklättrare (t.Ex. Murgröna - araliaceae)"
```
t.Ex. Murgröna pienillä

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -3125,29 +3147,29 @@ udk2 581.9 = "Geografisk botanik. Växtgeografin (fytogeografi). Floror. Växter
+udk2 582.093 = "Buskar (mindre vedartade växter utan central+f1393stam)"
+udk2 582.095 = "Undervegetation (t.Ex. Björnbär - rosoideae)"
+udk2 582.097 = "Trädklättrare (t.Ex. Murgröna - araliaceae)"
+udk2 582.099 = "Örtartade eller icke-vedartade växter (ofta kallade "Plantor" eller "Blommor")"
```
plantor ja blommor pienellä

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -3433,15 +3455,15 @@ udk2 62-9 = "Driftdata och -villkor"
 udk2 62-92 = "Reaktion i form av reaktanter. Reaktanter"
 udk2 62-93 = "Processer, maskiner och utrustning beroendee på hur de fungerar, kontinuitet, föreningar, miljö"
-udk2 62-94 = "Processernas riktning, hastighet, takt, längd, o.a.v."
+udk2 62-94 = "Processernas riktning, hastighet, takt, längd, o.A.V."
```
o.a.v. pienellä

EreMaijala replied:
vielä korjaamatta

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -3782,7 +3804,7 @@ udk2 682.1 = "Hovslageri"
 udk2 682.5 = "Möbelslag"
 udk2 682.6 = "Smide"
-udk2 683 = "Järnvaror. Hårdvaror. Låssmide. Förslutningar för flaskor o.dyj. Lamptillverkning. Uppvärmningsanordningar"
+udk2 683 = "Järnvaror. Hårdvaror. Låssmide. Förslutningar för flaskor o.Dyj. Lamptillverkning. Uppvärmningsanordningar"
```
Dyj pienellä

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -3833,7 +3855,7 @@ udk2 7.04 = "Ikonografi. Ikonologi. Konstnärliga motiv"
 udk2 7.06 = "Diverse frågor rörande konst"
 udk2 7.07 = "Konstbranschen.Expertis. Samlare"
-udk2 7.08 = "Karaktäristiska kännetecken, former, kombinationer (i konst, underhållning och sport)"
+udk2 7.08 = "Karaktäristiska kännetecken, former, kombinationer (I konst, underhållning och sport)"
```
I pienellä

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -3960,7 +3982,7 @@ udk2 793.5/.7 = "Sällskapsspel. Sällskapslek"
 udk2 793.7 = "Spel som inte kännetecknas av särskild handling"
 udk2 793.8 = "Vetenskapliga förströelser. Trolleri. Fingerfärdighet. Illusionism. Magiska trick"
-udk2 794 = "Bräd- och bordsspel (i form av tanke-, skicklighets- och turspel)"
+udk2 794 = "Bräd- och bordsspel (I form av tanke-, skicklighets- och turspel)"
```
I pienellä

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -4100,12 +4122,12 @@ udk2 811.214.21 = "Hindi"
 udk2 811.214.32 = "Bengali (bangla)"
 udk2 811.3 = "Döda språk av okänd grupptillhörighet. Kaukasiska språk"
-udk2 811.34 = "Döda språk av okänd tillhörighet, som talades omkring Medelhavet och Mellanöstern (utom semitiska språk)"
+udk2 811.34 = "Döda språk av okänd tillhörighet, som talades omkring medelhavet och mellanöstern (utom semitiska språk)"
```
medelhavet ja mellanöstern isolla

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -4194,9 +4216,9 @@ udk2 821.214 = "Modern indisk litteratur"
 udk2 821.214.32 = "Bengalisk (bangla) litteratur"
 udk2 821.3 = "Litteratur från döda språk av okänt grupptillhörighet. Kaukasisk litteratur"
-udk2 821.353 = "Litteratur från det sydliga (kartvelska) Kaukasus"
+udk2 821.353 = "Litteratur från det sydliga (kartvelska) kaukasus"
```
Kaukasus isolla

EreMaijala commented on local/languages/finna/classification/sv.ini:
```diff
@@ -4194,9 +4216,9 @@ udk2 821.214 = "Modern indisk litteratur"
+udk2 821.353 = "Litteratur från det sydliga (kartvelska) kaukasus"
 udk2 821.353.1 = "Georgisk litteratur"
-udk2 821.4 = "Afroasiatisk, Nilosahirska, Kongo-Kordofaniansk, Khoisan litteratur"
+udk2 821.4 = "Afroasiatisk, nilosahirska, kongo-kordofaniansk, khoisan litteratur"
```
Kongo isolla?

EreMaijala reviewed (changes requested):
Ymmärrän, että ainakin osa virheistä on jo lähtödatassa, mutta tehdään korjaukset ja jos rahkeet riittävät, niin lähetetään korjauspyynnöt myös lähtödatan ylläpitäjille. 

Kommenteissa "isolla" ja "pienellä" viittaa toki vain alkukirjaimeen.

karisera committed:
Virheiden korjausta ja parin luokan päivitystä

Korjasin Eren bongaamat virheet
udk2-luokkien käännöksiä tarkistettu Finton supistetusta UDK:sta http://finto.fi/udcs/fi/
Tuolta puuttuu tosiaan aika paljon noita udk2-luokkia, ja osa käännöksistä vaikuttaa vanhentuneilta. Toivottavasti ne voitaisiin jatkossa automaattisesti päivittää.

karisera committed:
Etydit

Etydit korjattiin pienellä

karisera committed:
Études pienellä

karisera committed:
Ruotsin luokka-käännösten paikkaamista

Samalla kun korjasin CAPS-otsikoita, kävi muutamia hassuja virheitä esim. i preposition muuttuessa isoksi alkukirjaimeksi (käytin englanninkielistä normalisoijaa, joka varmaan vaikutti). Nyt korjattu vastaavat fibat ja muutamat muut ongelmat isojen alkukirjainten kanssa.

karisera committed:
o.A.V. ==> o.a.v.
